requirements.txt

```diff
@@ -1,7 +1,9 @@
-pbr==1.8
+pbr==5.5.1 
 docker==3.7.3
-waiting==1.3.0
-requests==2.20.1
+waiting==1.3.0 ; python_version < '3.0'
+waiting ; python_version >= '3.0'
```

Comment:
Please set a version to python3 pips so we will know in all time what is use to build this commit

Reply:
Missed it, will fix.